string_permutation_problem/permute.py

```diff
@@ -0,0 +1,25 @@
+def permute(s):
+    out = []
+    
+    # Base Case
+    if len(s) == 1:
+        out = [s]
+        
```

Comment:
The code does not comply to PEP8.

*Origin: PEP8Bear, Section: `pep8`.*

The issue can be fixed by applying the following patch:

```diff
--- a/tmp/tmpb1s7hhaw/string_permutation_problem/permute.py
+++ b/tmp/tmpb1s7hhaw/string_permutation_problem/permute.py
@@ -4,7 +4,7 @@
     # Base Case
     if len(s) == 1:
         out = [s]
-        
+
     else:
         # For every letter in string
         for i, let in enumerate(s):
```